x-pack/test/apm_api_integration/tests/transactions/transactions_groups_statistics.ts

```diff
@@ -0,0 +1,325 @@
+            Object {
+              "x": 1607436780000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436840000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436900000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436960000,
+              "y": 0.02,
+            },
+            Object {
+              "x": 1607437020000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437080000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437140000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437200000,
+              "y": 0.02,
+            },
+            Object {
+              "x": 1607437260000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437320000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437380000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437440000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437500000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437560000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437620000,
+              "y": 0,
+            },
+          ]
+        `);
+        expect(item.errorRate.length).to.be.greaterThan(0);
+        expectSnapshot(removeEmptyCoordinates(item.errorRate)).toMatchInline(`
+          Array [
+            Object {
+              "x": 1607435880000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607435940000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436000000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436120000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436240000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436480000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436600000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607436960000,
+              "y": 0,
+            },
+            Object {
+              "x": 1607437200000,
+              "y": 0.5,
+            },
+          ]
+        `);
+      });
+      it('returns empty when transaction name is not found', async () => {
+        const response = await supertest.get(
+          url.format({
+            pathname: `/api/apm/services/opbeans-java/transactions/groups/statistics`,
+            query: {
+              start,
+              end,
+              uiFilters: '{}',
+              numBuckets: 20,
+              transactionType: 'request',
+              latencyAggregationType: 'avg',
```

Comment:
it'd be great to also have a very simple test for one of the percentile options, to ensure that doesn't silently break, also because we don't use it by default in the UI.